docs/src/routes/docs/components/dropdown-shell.svx

```diff
@@ -0,0 +1,98 @@
+---
+name: DropdownShell
+relatedComponents:
+  - name: Dropdown
+    link: ./docs/components/dropdown
+---
+
+<script>
+  import { DropdownShell, Dropdown, Button } from 'attractions';
+  import Showcase from 'src/containers/docs/showcase.svelte';
+  import ColorPreview from 'src/components/docs/color-preview.svelte';
+  import ShadowPreview from 'src/components/docs/shadow-preview.svelte';
+  import RelatedComponents from 'src/components/docs/related-components.svelte';
+
+</script>
+
+# DropdownShell
+
+The container for the [`Dropdown`](./docs/components/dropdown) activator and content.
+
+<RelatedComponents components={relatedComponents} />
+
+<Showcase>
+  <div slot="showcase" class="padded">
+    <DropdownShell let:toggle>
+      <Button on:click={toggle}>
+        Can contain more than one
+      </Button>
+      <Dropdown>
+        <div class="padded">
+          first
+        </div>
+      </Dropdown>
+      <Dropdown right>
+        <div class="padded">
+          second
+        </div>
+      </Dropdown>
+    </DropdownShell>
+    <DropdownShell>
+      Useless without a <code>&lt;Dropdown /&gt;</code>
+    </DropdownShell>
+  </div>
+  <div slot="source">
+
+```svelte
+<DropdownShell let:toggle>
+  <Button on:click={toggle}>
+    Can contain more than one
+  </Button>
+  <Dropdown>
+    <div class="padded">
+      first
+    </div>
+  </Dropdown>
+  <Dropdown right>
+    <div class="padded">
+      second
+    </div>
+  </Dropdown>
+</DropdownShell>
+<DropdownShell>
+  Useless without a <code>&lt;Dropdown /&gt;</code>
+</DropdownShell>
+```
+
+  </div>
+</Showcase>
+
+
+## Properties {#properties}
+
+### Functional Properties
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`open`** | `false` | `boolean` | Controls whether the dropdown content is shown or hidden. |
+
+### Class Properties
+
+All `<DropdownShell>` components use the `.dropdown-shell` class
+
+| Name | Default | Type | Description |
+| ---- | ------- | ---- | ----------- |
+| **`class`** | `null` | `string` | A class string to add to the underlying `<div>` element. |
+
+
+## Events {#events}
+
+| Name | Event Detail | Description |
+| ---- | ------------ | ----------- |
+| **`change`** | `value: boolean` | Fired when toggling the dropdown (with `value` being `true` for "open" and `false` for "closed"). |
+
+## Functions {#functions}
```

Comment:
New section, huh. I think it's best to name it `Exported Attributes` or `Exposed Attributes` and have the columns `Name | Type | Description`. Not all such attributes will be functions, and we need to accomodate for them. Arguments can be described in the description.

And in here I'd also like to mention that one would wanna use this attribute when toggling must be done in the same component where the shell is created. To put that in the description.

Also it might be worth adding under the header that those attributes can be accessed by using `bind:this` to a variable.

Reply:
I think a better way of stating the arguments (and return) types would be something similar to the Typescript way: `() => void` (omitting arg names as they are not relevant), but I'll stick with `Function` for now for consistency with the rest of the docs.

Reply:
Actually, we have some sort of internal inconsistency, because there's also the `async function*` as the type in the Autocomplete. Maybe we could have written something like AsyncGenerator, which, I believe, really is the type, but imo that one's very confusing. In this way, we should probably find a better way to specify functions. If the TS way would allow us to express async generators, then yes, that'd be the superior way